server/controllers/genAiController.ts

```diff
@@ -0,0 +1,57 @@
+import createAi from '@/server/utils/aiFactory';
+import { createError } from '@/server/middleware/errorHandling';
+
+import type { Request, Response, NextFunction } from 'express';
+import type { AiInterface, AiTypes } from '@/server/types/ai';
+
+const genAiController = {
```

Comment:
Consider using AI instead of ai or Ai throughout the codebase to improve consistency.

Reply:
I will change throughout, for consistency.